clang/include/clang/StaticAnalyzer/Core/PathSensitive/SymbolManager.h

```diff
@@ -101,8 +103,10 @@ class SymbolConjured : public SymbolData {
   }
 
 public:
-  /// It might return null.
-  const Stmt *getStmt() const { return S; }
```

Comment:
Would it make sense to keep `getStmt`? It could still return null but some callers might find it useful.